8Knot/pages/repo_overview/visualizations/ossf_scorecard.py

```diff
@@ -100,16 +109,42 @@ def ossf_scorecard(repo):
     # test if there is data
     if df.empty:
         logging.warning(f"{VIZ_ID} - NO DATA AVAILABLE")
-        return dbc.Table.from_dataframe(df, striped=True, bordered=True, hover=True)
+        return dbc.Table.from_dataframe(df, striped=True, bordered=True, hover=True), dbc.Label("No data")
 
     # repo id not needed for table
     df.drop(["repo_id"], axis=1, inplace=True)
 
+    # get all values from the data_collection_date column
+    updated_times = pd.to_datetime(df["data_collection_date"])
+
+    # we dont need to display this column for every entry
+    df.drop(["data_collection_date"], axis=1, inplace=True)
+
     df.loc[df.name == "OSSF_SCORECARD_AGGREGATE_SCORE", "name"] = "Aggregate Score"
     df.sort_values("name", ascending=True, inplace=True)
     df.rename(columns={"name": "Check Type", "score": "Score"}, inplace=True)
 
     table = dbc.Table.from_dataframe(df, striped=True, bordered=True, hover=True)
 
+    # format the date
+    try:
```

Comment:
Did you find a case where the scores for a single repo had a different collection date? To my knowledge this wont happen with how this data is collected

Reply:
Yup sounds good to me, in that case can this code be simplified?

Reply:
Can do! Do you want to keep the warning in there just in case the dates differ?

Reply:
What you have now looks good to me!